neptune/new/metadata_containers/metadata_container.py

```diff
@@ -274,6 +276,12 @@ def define(
         elif is_dict_like(value):
             value = Namespace(value)
         elif is_string_like(value):
+            simple_warning(
+                "You're assigning an object with an implicit cast to a string."
+                " Use `... = str(object)` instead."
+                " We'll end support of this behavior in `neptune-client==1.0.0`.",
+                stack_level=7,
```

Comment:
`define` jest teoretycznie metodą dostępną dla usera więc 7 brzmi dziwnie.

Reply:
It was based on my examples, especially this one:
```python
run['a1'] = A()
```